Neos.Flow/Classes/ResourceManagement/PersistentResource.php

```diff
@@ -323,6 +387,18 @@ public function setSha1($sha1)
      */
     public function getMd5()
```

Comment:
We removed the md5 part of PersistenceResource #2112

Reply:
So since this targets 5.3 (as it is a BUGFIX), we'll need to have an immediate follow-up for master (or carefully resolve the removal in the up-merge conflict then). Thanks for the heads-up! :)